networkx/generators/tests/test_expanders.py

```diff
@@ -63,6 +63,38 @@ def test_paley_graph(p):
             assert (v, u) in G.edges
 
 
+@pytest.mark.parametrize("d, n", [(2, 7), (4, 10), (4, 16)])
+def test_maybe_regular_expander(d, n):
+    """Test for the :func:`networkx.maybe_regular_expander` function."""
```

Comment:
We don't create docs for the tests, so these doc_strings aren't needed -- and the test names include this information, so I would prefer we just remove these test function doc_strings.